package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clay-utils",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
(This reflects what's on NPM; made a mistake with the release command when I last released)